packages/cadl-ranch-specs/http/special-words/main.tsp

```diff
@@ -1,84 +1,237 @@
 import "@typespec/http";
 import "@azure-tools/cadl-ranch-expect";
 import "@azure-tools/typespec-client-generator-core";
+import "./dec.js";
 
 using TypeSpec.Http;
 using Azure.ClientGenerator.Core;
 
-@doc("Illustrates operation, parameter and property have name of special words or characters.")
+/**
+ * Scenarios to verify that reserved words can be used in service and generators will handle it appropriately.
+ *
+ * Current list of special words
+ * ```txt
+ * and
+ * as
+ * assert
+ * async
+ * await
+ * break
+ * class
+ * constructor
+ * continue
+ * def
+ * del
+ * elif
+ * else
+ * except
+ * exec
+ * finally
+ * for
+ * from
+ * global
+ * if
+ * import
+ * in
+ * is
+ * lambda
+ * not
+ * or
+ * pass
+ * raise
+ * return
+ * try
+ * while
+ * with
+ * yield
+ * ```
+ */
 @scenarioService("/special-words")
 namespace SpecialWords;
 
-@route("/operation")
-@doc("This interface is for testing operations has a special name.")
+/**
+ * Test reserved words as operation name.
+ */
+@route("/operations")
 @operationGroup
-interface Operation {
-  @scenario
-  @scenarioDoc("A operation name of `for` should work.")
-  @get
-  @route("/for")
-  for(): void;
+interface Operations {
+  @opNameScenario("and") and(): void;
```

Comment:
qq: what is `@opNameScenario` vs `@scenario`?

Reply:
yeah just made some simple decorators so this spec doesn't look like a big mess as everything is about the same x100